src/getters.jl

```diff
@@ -1885,4 +1885,90 @@ function GetDatabaseCohorts(
     
 end
 
-export GetDatabasePersonIDs, GetPatientState, GetPatientGender, GetPatientRace, GetPatientAgeGroup, GetPatientVisits, GetMostRecentConditions, GetMostRecentVisit, GetVisitCondition, GetPatientEthnicity, GetDatabaseYearRange, GetVisitPlaceOfService, GetVisitConcept, GetVisitDate, GetDrugExposures, GetDrugConceptIDs, GetDrugAmounts, GetVisitProcedure, GetDatabaseCohorts, GetCohortSubjects, GetCohortSubjectStartDate, GetCohortSubjectEndDate, GetDrugExposureIDs
+"""
+function GetDrugExposureEndDate(drug_exposure_ids; tab = drug_exposure)
+
+Given a list of drug_exposure IDs, find their corresponding drug_exposure_end_date ID.
```

Comment:
I'd say instead:

"Given a list of drug_exposure IDs, find their exposure end dates."